integration/integration_test.go

```diff
@@ -128,6 +135,10 @@ func TestMain(m *testing.M) {
 }
 
 func TestAuditSeccomp(t *testing.T) {
+	if *deploySPOFlag == false {
+		t.Fatal("The Security Profiles Operator was not deployed. Please check that the deploy-spo flag is true")
+	}
+
```

Comment:
I would prefer to let it fail if the SPO is not already installed in the system, don't you think?

Reply:
Yeah, I think it makes sense to let it fail.